### DISCUSSION TITLE
自己随手加了一个随机图片API

### DISCUSSION THREAD

**Discussion by shiker1996** [halo-dev/halo#2463] [Ideas]
很简单的一个功能, 这个效果示意如下：
![image](https://user-images.githubusercontent.com/32234876/192083936-e76fbc18-b541-4372-9dc1-d87bb3c95df0.png)

可以通过修改home下的.halo中的img.txt, 修改随机图片地址
![image](https://user-images.githubusercontent.com/32234876/192083978-1ad3065c-b5a2-458e-97d2-c182e8ed6e23.png)

打游戏间隙做出来的😂

**Comment by shiker1996** [comment]
参考php的随机图片api搭的，参考地址：https://blog.csdn.net/woodwhale/article/details/118832228?utm_medium=distribute.pc_relevant.none-task-blog-2~default~baidujs_baidulandingword~default-1-118832228-blog-121585039.pc_relevant_multi_platform_whitelistv3&spm=1001.2101.3001.4242.2&utm_relevant_index=4
自己搭完的随机图片API地址：https://shiker.tech/randomPic

**Comment by ruibaby** [comment]
不错。或许也可以结合 2.0 的插件系统来做一些事情，参考 https://github.com/halo-sigs/plugin-unsplash

**Reply by shiker1996** [reply]
> 不错。或许也可以结合 2.0 的插件系统来做一些事情，参考 https://github.com/halo-sigs/plugin-unsplash

可以的，我看最近2.0已经修复了mysql的支持，抽空我看下
